docs/guides/cloudeventssource.md

```diff
@@ -0,0 +1,120 @@
+# Receiving External CloudEvents
+
+The Trigermesh `CloudEventsSource` API object is used to ingest Cloudevents produced from external sources.
+
+## Configuring a CloudEventsSource Object
+
+The CloudEventsSource accepts parameters to set authentication, URL path and rate limiter. When succesfuly created it exposes an endpoint to listen for CloudEvents.
+
+### Configuring Credentials (Optional)
+
+Credentials can be configured using [Basic Authentication](https://datatracker.ietf.org/doc/html/rfc2617) using Kubernetes secrets to manage passwords.
+
+Credentials are defined as arrays, allowing clients to use multiple user/password items.
+
+The credentials are defined under `spec.credentials.basicAuths`:
+
+```yaml
+spec:
+  credentials:
+    basicAuths:
+    - username: user1
+      password:
+        valueFromSecret:
+          name: password1
+          key: password
+    - username: user2
+      password:
+        valueFromSecret:
+          name: password2
+          key: password
+```
+
+### Configuring Path (Optional)
+
+The `spec.path` parameter is used configure the URL path where CloudEvents will be accepted.
+
+```yaml
+spec:
+  path: /mypath
+```
+
+### Configuring Rate Limiter (Optional)
+
+Rate Limiter is used to filter the quantity of requests per second that an adapter instance can receive. When the configured limit per time window is reached, HTTP code 429 is returned along information on when the client should retry.
+
+```console
+HTTP/1.1 429 Too Many Requests
+retry-after: 1650204469582011930
+```
+
+To configure the Rate Limiter use the `spec.rateLimiter.requestsPerSecond` parameter:
+
+```yaml
+spec:
+  rateLimiter:
+    requestsPerSecond: 1000
+```
+
+It must be noted when configuring the Rate Limiter that:
```

Comment:
In the shoes of a user, when I read these two points I might be concerned about the impact this could have on my project. Do we want to point to a solution or maybe just remove these for now if we don't know how to address them? It also is showing a bit of "under the hood" stuff.

Reply:
Removed the Knative references and moved the note to a dedicated  `tip` section.
32c4c7a5d30aa479098271d4ebf39b40a4462c20